src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  font-weight: 500;
+  font-size: 12px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding-left: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-transform: uppercase;
+
+  &__logo {
+    height: 40px;
+  }
+
+  .nav {
+    margin-right: 50px;
+    display: flex;
+    height: 60px;
```

Comment:
It is probably better to let the height be defined by the items 
(BTW - you are already doing that and can just remove `.nav height`)